server/routes/teacherPrivilege.js

```diff
@@ -17,6 +17,9 @@ router.post('/api/v1/users/update', checkIfUserHasSchool, checkUserUpdatePrivile
 // adds new result to the database
 router.post('/api/v1/results/addresults', checkIfUserHasSchool, AddResultsValidation.validateAddResults, ResultsController.addResults);
 
+// update result in the database
+router.post('/api/v1/results/update', checkIfUserHasSchool, checkResultUpdatePrivilege, AddResultsValidation.validateUpdateResults, ResultsController.updateResult);
```

Comment:
@jamsgra I do not think the appropriate HTTP verb to be used here is `post`. According to the resource at https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods, the appropriate HTTP verb should be either PUT or PATCH.

Reply:
@sulenchy you are right, I have pushed changes to use PUT for both `updateUser` & `updateResult`

Reply:
@sulenchy Actually looking at it, PATCH may be the more suitable option as the entire record isn't necessarily updated. What do you think?